.vib/redis-cluster/goss/goss.yaml

```diff
@@ -1,39 +1,50 @@
+{{- $auth := printf "REDISCLI_AUTH=%s" .Vars.password }}
+{{- $headless_port := .Vars.redis.containerPorts.redis }}
```

Comment:
It took me some time to understand the tests where this variable was involved because of its naming. It is used both in connections that use the headless svc and plain IPs:

- `-h redis-cluster-{{ $i }}.redis-cluster-headless -p {{ $headless_port }}`: OK, it is easy to understand as the hostname is a `headless` svc.
- `xargs -I '{}' redis-cli -c -h '{}' -p {{ $headless_port }}`: Not that straightforward to notice :(, the host is an IP

I would simplify to something like `redis_port`, which IMHO makes it easier to understand.

Reply:
Applied both suggestions